### PR TITLE
Network Type missing

### PR DESCRIPTION
**Pull request by razi-ahmad1**

### CODE REVIEW

File: src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy
Comment (ddeyoung) [inline src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy:1]: need dockerBridge, host, overlay, and xenNetwork to be returned here.

**Review by ddeyoung — CHANGES_REQUESTED** [review]

File: src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy
Comment (razi-ahmad1) [reply src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy:1]: @ddeyoung code updated as suggested.

Comment (ddeyoung) [reply src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy:1]: xenNetwork needs created here.

**Review by ddeyoung — CHANGES_REQUESTED** [review]

File: src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy
Comment (razi-ahmad1) [reply src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy:1]: @ddeyoung changes has been done as suggested

**Pull request merged**